front-end/src/components/CreateEditPostModal.tsx

```diff
@@ -117,15 +133,32 @@ export default function CreateEditPostModal(props: Props){
           }).catch((error: any) => {
             setShowError(true)
           })
+
+        //put to post_id
+        const uuid = generateUUID();
+        console.log(uuid)
+        AxiosWrapper.put(process.env.REACT_APP_API_URL + "/api/author/" + props.loggedInUser.authorId + "/posts/" + uuid + "/", data, props.loggedInUser)
```

Comment:
please use a template string here instead of concatenating strings. For example, `${process.env.REACT_APP_API_URL}/api/author/${props.loggedInUser.authorId}/posts/${uuid}/`